src/components/inputs/DeleteButton/DeleteButton.tsx

```diff
@@ -15,6 +18,14 @@ function DeleteButton({ event, onDelete }: DeleteButtonProps) {
 
   const isNotOwner = account && event.owner?.id !== account.localAccountId;
 
+  const { isShown, toggle } = useModal();
+
+  const onConfirm = () => {
+    handleDelete();
+    toggle();
+  };
+  const onCancel = () => toggle();
```

Comment:
Unnecessary variable, but if you want to keep it for readbility purposes then the following also works:
```suggestion
  const onCancel = toggle;
```